src/interfaces/IMetaMorpho.sol

```diff
@@ -137,9 +137,8 @@ interface IMetaMorphoBase {
     /// increase the cost of depositing to the vault.
     function setSupplyQueue(Id[] calldata newSupplyQueue) external;
 
-    /// @notice Sets the withdraw queue as a permutation of the previous one, although markets with both zero cap and
-    /// zero vault's supply can be removed from the permutation.
-    /// @notice This is the only entry point to disable a market.
+    /// @notice Sets the withdraw queue as a permutation of the previous one, although some markets can be removed.
+    /// @notice This is the only entry point to remove a market.
```

Comment:
I still find this comment misleading (#362) even if I understand what it means

Reply:
What's your prop?

Reply:
remove it

Reply:
```suggestion
```